src/main/webapp/itempagescript.js

```diff
@@ -145,13 +145,13 @@ function getItemPageComments(comments) {
  * @param { boolean } belongsToUser - boolean that shows whether or not the current user has posted a comment
  * @returns { html element }  returns a list element
  */
-function createListElement(comment, belongsToUser) {
+function createListElement(comment, belongsToUser, email) {
   const liElement = $('<li class="list-group-item"></li>');
   liElement.text(comment);
   if (belongsToUser) {
-    liElement.append($('<i class="fa fa-trash" style="float:right;"></i>'));
+    liElement.append($('<i class="fa fa-trash" style="float:right;"></i>')); 
   }
-
+    liElement.append($('<img class="pr-1" src="'+ "https://icotar.com/avatar/"+ email.charAt(0) +".png?s=23" + '"style=float:left;"></img>'));
```

Comment:
Apply formatting to this file because I think this line might be over 80 characters long.